packages/server/src/queries/complex/concentrated-liquidity/index.ts

```diff
@@ -230,20 +230,16 @@ export type UserPositionDetails = Awaited<
 /** Appends user and position details to a given set of positions.
  *  If positions are not provided, they will be fetched with the given user address. */
 export async function mapGetUserPositionDetails({
```

Comment:
I traced where this is being used. The caller simply fetches positions then passes them to this function. What if we query for positions in this function then call it `getUserPositionsDetails`?

I sense, that everywhere this function would be called, the caller would first have to fetch positions before calling this function. Moving that fetch here would slightly simplify things, but it could cause repeated querying of positions if other data is needed that is not returned by this function.

To get around that though, we could also return the original positions from that query in this function and that data would not have to be requeried in that case.